File: JAVApb5/src/main/java/concurs/domain/ProbaDTO.java
package concurs.domain;

public class ProbaDTO extends  Proba{
    private int nrParticipanti;


    public ProbaDTO(String denumire, int varstaMin, int varstaMax,int nrParticipanti) {
        super(denumire, varstaMin, varstaMax);
        this.nrParticipanti=nrParticipanti;
    }

    public int getNrParticipanti() {
        return nrParticipanti;
    }

    public void setNrParticipanti(int nrParticipanti) {
        this.nrParticipanti = nrParticipanti;
    }
}

File: CSharp/domain/ProbaDTO.cs
namespace CSharp.domain
{
    public class ProbaDTO:Proba
    {
        public int nrParticipanti{ get; set; }

        public ProbaDTO(string denumire, int varstaMin, int varstaMax, int nrParticipanti) : base(denumire, varstaMin, varstaMax)
        {
            this.nrParticipanti = nrParticipanti;
        }

        public ProbaDTO(int nrParticipanti)
        {
            this.nrParticipanti = nrParticipanti;
        }

        public override bool Equals(object obj)
        {
            return base.Equals(obj);
        }

        public override int GetHashCode()
        {
            return base.GetHashCode();
        }

        public override string ToString()
        {
           // return base.ToString();
            return string.Format("[Proba: Id={0}, Denumire={1},VarstaMin={2},VarstaMax={3},nrParticipanti={4}]", Id, Denumire,VarstaMin,VarstaMax,nrParticipanti);
        }
    }
}